CorsixTH/Lua/game_ui.lua

```diff
@@ -151,12 +153,16 @@ function GameUI:draw(canvas)
     canvas:fillBlack()
   end
   local zoom = self.zoom_factor
+  local dx = self.screen_offset_x +
+      math.floor((0.5 - math.random()) * self.shake_screen_intensity * 100)
+  local dy = self.screen_offset_y +
+      math.floor((0.5 - math.random()) * self.shake_screen_intensity * 100)
```

Comment:
Instead of magic 100, why not make the shake_screen_intensity have tile units, and do the translation from intensity to tiles in the beginShakeScreen?